action.yml

```diff
@@ -56,6 +56,7 @@ runs:
         workflow_inputs: '{
           "account": "${{ inputs.account_id }}",
           "branch": "${{ inputs.branch }}",
+          "infra_live_repo": "${{ vars.GITHUB_REPOSITORY }}",
```

Comment:
Should these be `inputs` instead of `Vars`. I also do not see an infra-live PR that passes this as input or sets the variable. Am i missing some context?

Reply:
This is set automatically by github